src/utils/useDocumentTitle.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+
+import playroomConfig from '../config';
+
+interface UseDocumentTitleProps {
+  title?: string;
+  suffix?: string;
+}
+
+export function useDocumentTitle({
+  title,
+  suffix,
+}: UseDocumentTitleProps): void {
```

Comment:
Declaring return types, particularly `void`, is not really a practice we have employed in the repo. Suggest keeping things consistent.

```suggestion
}: UseDocumentTitleProps) {
```